### PR TITLE
[18088] CLI: participant user transports parser

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**

### CODE REVIEW

File: cli/test/README.md [hunk 1]
```diff
@@ -93,8 +93,9 @@ This document includes CLI commands and the expected CLI output in order to help
...
 |`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[].address`|`ERROR: set command for Participant builtin <external_locators> expects at least 1 arguments and received 0`|`PARTICIPANT_BUILTIN_METATRAFFIC_EXTERNAL_UNICAST_LOCATORS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[].address -h`|N/A|`PARTICIPANT_BUILTIN_METATRAFFIC_EXTERNAL_UNICAST_LOCATORS_USAGE`|
-|`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[].address argument` [^2]|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
-|`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[11811].address argument`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.external_locators.metatraffic_unicast[].address argument` [^2]|N/A|N/A|
```
Comment (JesusPoderoso) [inline cli/test/README.md:96]: Why is `[^2]` added to a `set` command?
Isn't it refer only to `print`, `clear` and `query`?

File: cli/src/cpp/usages.hpp [hunk 2]
```diff
@@ -292,6 +292,33 @@ static const char PARTICIPANT_USE_BUILTIN_TRANSPORTS_USAGE[] =
...
+      fastddsqosprof <file> <command> participant[<profile_name>].user_transports[(<index>)] [<transport_id> (help | -h | --help)]
+      fastddsqosprof <file> <command> participant[<profile_name>].user_transports[(<index>)] (help | -h | --help)
+
+    Options:
+      help -h --help    CLI participant user transports usage
+
+    The allowed <command> options are:
+      clear             Erase user transport from the participant list.
+      print             Print user transport in the participant list.
+      set               Add or update user transport in the participant list.
```
[the remark this answers is not in the document]
Comment (JLBuenoLopez-eProsima) [reply cli/src/cpp/usages.hpp:308]: The query has another usage:

```
fastddsqosprof <file> query (<keys | size>) element
```

Consequently, another usage should be defined for QUERY commands. That is the reason in the code sometimes a QUERY_USAGE is expected and marked as TODO.

File: cli/test/README.md (hunk 1 above)
Comment (JLBuenoLopez-eProsima) [reply cli/test/README.md:96]: That is a footnote just to remember that in those verbs, an additional validity check will be needed. It is a TODO in the test plan, so to say.

Comment (JesusPoderoso) [reply cli/test/README.md:96]: Nice!

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Pull request merged**